src/ontology/imports/envo_terms.txt

```diff
@@ -1,2 +1,5 @@
-ENVO:00001998
-
+http://purl.obolibrary.org/obo/ENVO_00001998
+http://purl.obolibrary.org/obo/ENVO_00000446
+http://purl.obolibrary.org/obo/ENVO_00000447
+http://purl.obolibrary.org/obo/ENVO_00002030
+http://purl.obolibrary.org/obo/ENVO_01000048
```

Comment:
Why is this needed? Does it even work?

Reply:
(in particular since you deleted all the other term files)

Reply:
The terms files that I deleted are from the imports that are not in the config file.

Reply:
Ok cool.